code/modules/mining/machine_vending.dm

```diff
@@ -34,6 +34,7 @@
 		new /datum/data/mining_equipment("Fulton Pack",			/obj/item/weapon/extraction_pack,										1000),
 		new /datum/data/mining_equipment("Lazarus Injector",	/obj/item/weapon/lazarus_injector,										1000),
 		new /datum/data/mining_equipment("Silver Pickaxe",		/obj/item/weapon/pickaxe/silver,										1000),
+		new /datum/data/mining_equipment("Luxury Shelter Capsule",	/obj/item/weapon/survivalcapsule/luxury,							1500),
```

Comment:
yeah 1500 is definitely a huge deal for two free medkits and a huge space with air, alone two brute medkits are 1200, and normal capsules are 400. I'd recommend at least 2000 or 3000 even for having a free base anywhere that's large enough to serve as a decent cult base